docs/ref/models/indexes.txt

```diff
@@ -57,6 +57,22 @@ The name of the index. If ``name`` isn't provided Django will auto-generate a
 name. For compatibility with different databases, index names cannot be longer
 than 30 characters and shouldn't start with a number (0-9) or underscore (_).
 
+``db_tablespace``
+-----------------
+
+.. attribute:: Index.db_tablespace
+
+.. versionadded:: 2.0
+
+The name of the :doc:`database tablespace </topics/db/tablespaces>` to use for
+this index. For single field indexes, if ``db_tablespace`` isn't provided, the
+index is created in the ``db_tablespace`` of the field.
+
+If :attr:`~django.db.models.Field.db_tablespace` isn't specified (or if the
```

Comment:
I think we should use:
```
:attr:`.Field.db_tablespace`
```
here so that the "Field" part appears in the rendered docs.